DuckDuckGo/NetworkProtectionVPNSettingsView.swift

```diff
@@ -127,4 +136,18 @@ struct NetworkProtectionVPNSettingsView: View {
 
 }
 
+private extension WidgetEducationView {
+    static var vpn: Self {
+        WidgetEducationView(
+            navBarTitle: UserText.vpnSettingsAddWidget,
+            thirdParagraphText: UserText.addVPNWidgetSettingsThirdParagraph,
+            widgetExampleImageConfig: .init(
+                image: Image("WidgetEducationVPNWidgetExample"),
+                maxWidth: 164,
+                horizontalOffset: -7
```

Comment:
This is gross but I couldn’t think of another way (it’s because the assets have shadow which differ…)